frontend/react/answers/samcrider.md

```diff
@@ -0,0 +1,23 @@
+## QUESTIONS
+
+### What is a prop?
+
+Props are the information that you pass to a JSX tag. For example: className and src. Components can pass props down to their children.
+
+### What is a hook? Can you explain what the built in hooks are, and when you would use them?
+
+Hooks are JS functions that employ stateful logic. React comes with several types of built-in hooks. These include state, context, ref, and effect hooks. State hooks store user input and that input can be recalled easily here. I would use a stateful hook to track an index because I could directly manipulate the index using useState. Context hooks allow for information to be passed down from a distant parent without using props. Ref hooks hold values that aren't rendered. These are important for working with non-react systems like the DOM. Finally, effect hooks are used to connect a react componen with an external system. An example would be a useEffect hook that performs an action upon an event being triggered.
```

Comment:
effect hooks are used to connect a react component*

I think a better definition for useEffect is it allows components to be responsive to changes in specified dependencies, in other words it allows components to "watch and respond" to certain events.